server/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM nginx:1.15.1-alpine
+
+COPY default.conf /etc/nginx/conf.d/default.conf
+
+COPY start.sh /
```

Comment:
I would create a init container to avoid using a custom image

Reply:
yep. will do!

Reply:
done, please check it out